api/src/routes/auth.ts

```diff
@@ -23,6 +23,7 @@ const defaultUser = {
   completedChallenges: [],
   currentChallengeId: '',
   emailVerified: false,
+  externalId: 'hi',
```

Comment:
```suggestion
  externalId: '',
```
We'll need rework this into a real default user at some point, but `''` seems fine for now.